src/main/java/it/cnr/isti/hpc/wikipedia/article/Article.java

```diff
@@ -53,18 +53,19 @@
 	private String timestamp;
 	private Type type = Type.ARTICLE;
 	private String enWikiTitle;
-	private List<Table> tables;
-	private List<Link> images;
+	private transient List<Table> tables;
+	private transient List<Link> images;
 	protected List<List<String>> lists;
 	private List<Link> links;
 	private List<Link> externalLinks;
 	protected String redirect;
-	private List<String> sections;
-	private List<String> paragraphs;
-	private List<Link> categories;
-	private List<Template> templates;
-	private List<String> templatesSchema;
-	private List<String> highlights;
+	private transient List<String> sections;
+	private transient List<String> paragraphs;
```

Comment:
I think paragraphs might be needed, since we might want to create 'fake links' on text to get more reliable stats for entity linking, what do you reckon @naveenmadhire @dav009 ?